### DISCUSSION TITLE
pods created by argo workflows causing: No scale down because workflow pod not backed by a controller

### DISCUSSION THREAD

**Discussion by life0215** [argoproj/argo-workflows#10494] [General]
Is there any other options to avoid this scaling issue other than adding below annotation to workflow podMetadata.
```cluster-autoscaler.kubernetes.io/safe-to-evict: "true"```
<img width="1718" alt="image" src="https://user-images.githubusercontent.com/16345094/217973604-0b063b16-205e-4b26-9afc-f3231af2c645.png">

**Comment by tooptoop4** [comment]
did u solve? I get "pod deleted" a lot
